src/Components/DropdownItem/DropdownItem.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  handleItemClick: (person: Person) => void,
+};
+
+export const DropdownItem: React.FC<Props> = ({
+  person,
+  handleItemClick,
+}) => {
+  return (
+    <div
+      className="dropdown-item"
+      role="button"
+      style={{ cursor: 'pointer' }}
+      tabIndex={0}
+      onKeyDown={() => {}}
+      onClick={() => handleItemClick(person)}
+    >
+      <p className={classNames({
+        'has-text-link': person.sex === 'm',
+        'has-text-danger': person.sex === 'f',
+      })}
+      >
+        {person.name}
```

Comment:
when you hover on names in dropdown it shoud be `cursor pointer`,
you have to show a user that this item is clickable

you can use `a` tag instead of `div` or add css styles
![image](https://github.com/mate-academy/react_autocomplete/assets/91946036/cb714938-7f81-42ae-b224-e2737617549d)